sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -804,6 +820,10 @@
                 -snippetpath ${project.basedir}/sdk/core/azure-core/src/samples/java
                 -snippetpath ${project.basedir}/sdk/core/azure-core-http-netty/src/samples/java
                 -snippetpath ${project.basedir}/sdk/core/azure-core-http-okhttp/src/samples/java
+                -snippetpath ${project.basedir}/sdk/core/azure-core-serializer-avro-apache/src/samples/java
+                -snippetpath ${project.basedir}/sdk/core/azure-core-serializer-avro-jackson/src/samples/java
+                -snippetpath ${project.basedir}/sdk/core/azure-core-serializer-json-gson/src/samples/java
+                -snippetpath ${project.basedir}/sdk/core/azure-core-serializer-json-jackson/src/samples/java
```

Comment:
Remove these as well.

Reply:
Talked offline, decided it is safe to leave as it won't affect anything.